content/en/blog/2025/expose-otel-collector-gateway-api/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
```

Comment:
```suggestion
title: Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
linkTitle: tbd
```

Please come up with a short title, that will be used for the link and in the navigation. Most titles do only have a few words